ccmlib/scylla_node.py

```diff
@@ -1571,9 +1580,22 @@ def run_scylla_sstable(self, command, additional_args=None, keyspace=None, dataf
             additional_args = []
 
         scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
-        sstables = self._Node__gather_sstables(datafiles, keyspace, column_families)
         ret = {}
 
+        if datafiles is None and keyspace is not None:
+            tag = "sstable-dump-{}".format(uuid.uuid1())
+            kts = ",".join([f"{keyspace}.{column_family}" for column_family in column_families])
```

Comment:
No need to materialize list
```suggestion
            kts = ",".join(f"{keyspace}.{column_family}" for column_family in column_families)
```

Reply:
Done in v2.